bc_obps/reporting/models/report_operation.py

```diff
@@ -49,6 +49,12 @@ class ReportOperation(TimeStampedModel):
         blank=True,
         null=True,
     )
+    registration_purpose = models.CharField(
+        max_length=1000,
+        db_comment="The Registration purpose of the operation",
+        blank=True,
+        null=True,
```

Comment:
The registration purpose shouldn't be allowed to be null. We won't have any data in the db by the time this is deployed, so you can give django any value when you run `make migrations`